persistence/genesis.go

```diff
@@ -145,17 +145,14 @@ func (m *persistenceModule) populateGenesisState(state modules.PersistenceGenesi
 	}
 
 	// Updates all the merkle trees
-	appHash, err := rwContext.ComputeAppHash()
+	_, err = rwContext.ComputeStateHash()
```

Comment:
I would probably at least log the stateHash for debugging purposes.

Any thoughts against that?

Reply:
Potentially addressed here: [`39da0ed` (#391)](https://github.com/pokt-network/pocket/pull/391/commits/39da0ed1ab5e3dc8ac1101d2901fe8b752e5b7a2)